packages/tooltip-extension/src/index.ts

```diff
@@ -283,6 +283,7 @@ namespace Private {
    */
   export function fetch(options: IFetchOptions): Promise<JSONObject> {
     const { detail, editor, kernel } = options;
+
```

Comment:
If this is a manual changes, let's remove this change to avoid confusion as tooltip is not modified by this PR